readme.md

```diff
@@ -71,13 +90,21 @@ kept out of the trigger and be put in domain classes.
 
 ## Principles
 
+In order to make clean code, use the following principles of SOLID:
+- SRP: Single Responsibility Principal
+- OCP: Open/Close Principal
+- LSP: Liscov Substitution Principal
+- ISP: Interface Segregation Principle
+- DIP: Dependency Inversion Principle
```

Comment:
Could you add a line or two to just quickly note what these principles are about?